tools/run-container

```diff
@@ -351,9 +351,8 @@ wait_for_boot() {
         if [ "$OS_NAME" = "centos" ]; then
             debug 1 "configuring proxy ${http_proxy}"
             inside "$name" sh -c "echo proxy=$http_proxy >> /etc/yum.conf"
-            inside "$name" sed -i s/enabled=1/enabled=0/ \
-                /etc/yum/pluginconf.d/fastestmirror.conf
-            inside "$name" sh -c "sed -i '/^#baseurl=/s/#// ; s/^mirrorlist/#mirrorlist/' /etc/yum.repos.d/*.repo"
+            inside "$name" sh -c "sed -Ei '/^#baseurl=/s/#// ; /^(mirrorlist|metalink)=/s/^/#/' /etc/yum.repos.d/*.repo"
+            inside "$name" sh -c "sed -i 's/download\.fedoraproject\.org/dl.fedoraproject.org/g' /etc/yum.repos.d/*.repo"
```

Comment:
s/-E/--regexp-extended/

Reply:
I assume you are suggesting this to follow the general good practice of using long options in scripts. I'm fine with switching to the long format, but I think `sed -E` is a bit special as it's the most widely used for to ask for extended regexp.

Reply:
yes that was why.  -E is maybe special in your head, but I had to go to a man page.

Reply:
@smoser `-E` is a bit special in that extended regexps are a POSIX thing and `sed -E` is the POSIX way to enable them, while `--regexp-extended` is a GNU sed thing. This said, we're not tagetting POSIX and I like the long version. /paride --enable-verbose-mode :)

Reply:
If --regexp-extended doesn't work, then using -E is acceptable (with a comment as to what that means).  I'm not hung up on it though.

I'll mark 'approved' and you can change that to your liking.